app/helpers/language_helper.py

```diff
@@ -35,8 +35,8 @@ def handle_language(metadata: Optional[MetadataProxy] = None) -> None:
                 schema = load_schema_from_metadata(
                     metadata=metadata, language_code=request_language
                 )
-                if schema.json["title"] != cookie_session.get("survey_title"):
-                    cookie_session["survey_title"] = schema.json["title"]
+                if schema.json["title"] != cookie_session.get("title"):
```

Comment:
Can we not keep the 'survey_title' instead of  changing  it to 'title'? What is the motivation for this change?

Reply:
This is the title in the cookie and has a different name to differentiate from the `survey_title` in the survey config